### PR TITLE
Beta

### PR DESCRIPTION
**Pull request by pandabear189**
**FILL OUT THE FIELDS BELOW!**

**Discord Username and Tag**
Ex: Space#9878

**Describe the Changes You've Created**

**Where are these changes going?**
Specific Sub-Servers or the Main Server

**Confirm that you have done the following:**
- [ ] Created a new folder for your bot under utils/bots/~
- [ ] Moved any events under the events folder. 
- [ ] Properly Documented the code changes
- [ ] DM'd Space for a new Documentation Page (for new commands)
- [ ] Tested the New Files on your own and you confirm that the changes are bug free.

**Additional Context**
Add any other information about the Pull Request here.

### CODE REVIEW

File: core/checks.py [hunk 1]
```diff
@@ -1,158 +1,122 @@
...
-
-    return any(role.id in rolesID for role in ctx.author.roles)
-
-
-is_mktCommissionAuthorized = commands.check(mktCommissionAdd)
+    return app_commands.check(predicate)
 
 
-def TimmyBetaHost(ctx):
+def timmy_beta_host(ctx):
```
Comment (Space-Turtle0) [inline core/checks.py:119]: Requires inner predicate function?

**Review by Space-Turtle0 — CHANGES_REQUESTED** [review]

File: core/checks.py (hunk 1 above)
Comment (pandabear189) [reply core/checks.py:119]: I think puncher may know (im not sure :/)

File: core/common.py [hunk 2]
```diff
@@ -20,32 +17,34 @@
...
 load_dotenv()
 
-# module variables
+# Module Variables
 coroutineType = Callable[[Any, Any], Awaitable[Any]]
 g = Github(os.getenv("GH_TOKEN"))
 
 
-class ConfigcatClient:
+class ConfigCatClient:
```
Comment (Space-Turtle0) [inline core/common.py:47]: The change to this class name did not get reflected everywhere, as indicated by this [failed travis build](https://app.travis-ci.com/github/School-Simplified/Timmy-SchoolSimplified/jobs/568014227).

**Review by Space-Turtle0 — CHANGES_REQUESTED** [review]

File: core/checks.py (hunk 1 above)
Comment (Space-Turtle0) [reply core/checks.py:119]: Hm alright, I thought it was a normal check at first but then it was placed in the `commands.check` function so I'm not sure if that's intentional or not. I'll check the documentation real quick.

File: core/common.py (hunk 2 above)
Comment (pandabear189) [reply core/common.py:47]: i will change these tmrw, but ill double check with puncher since he made many changes

File: core/checks.py (hunk 1 above)
Comment (Space-Turtle0) [reply core/checks.py:119]: Never mind it's fine

**Pull request closed without merging**